apps/tallycoin-connect/data/tallycoin_api.key

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file needed if it's empty?

Reply:
I think it couldn't get written if it wasn't present already -- iirc that's the reason why I added this empty version.

Reply:
Ahh got it. Where you able to connect the app with the server ok? I seem to be having issues.